src/renderer/chunk_meshing.zig

```diff
@@ -633,6 +633,7 @@ pub const ChunkMesh = struct {
 			.isStillNeeded = @ptrCast(&isStillNeeded),
 			.run = @ptrCast(&run),
 			.clean = @ptrCast(&clean),
+			.taskType = .lighting,
```

Comment:
Also I just noticed you have three .lighting tasks, but they do completely different things.
Having multiple tasks with the same tag can hide performance problems. For example, this LightRefreshTask is relatively fast, which can skew the average mesh generation time.

Reply:
It wouldn't be hard to change. Is there a better classification? I literally just classified it based on my impressions of the performance problems.

Reply:
I would just put the LightRefreshTaks and the LightMapLoadTask under .misc, they shouldn't take much performance.